mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java

```diff
@@ -73,6 +73,8 @@
     public static final String ERROR_FINDING_VPA = "Error finding Virtual Payment Address";
     public static final String ERROR_FINDING_MOBILE_NUMBER = "Error finding Mobile Number";
     public static final String PLEASE_ENTER_VALID_AMOUNT = "Please enter a valid amount";
+    public static final String PLEASE_ENTER_VALID_VPA = "Please enter a valid VPA";
```

Comment:
Instead of declaring inside `Constants.java` please create a _string resource_. Since this message will be displayed to the user, so if in the future we plan to provide multi-language support then this approach will not work.

Reply:
@devansh-299 I've updated my PR . Please review again.